src/components/Link.tsx

```diff
@@ -253,43 +258,41 @@ export function InlineLink({
   const flattenedStyle = flatten(style)
 
   return (
-    <TouchableWithoutFeedback
-      accessibilityRole="button"
+    <Text
```

Comment:
Tbh idk why I had this wrapped in a touchable, but UITextView didn't like it since it's not Text. No functional changes here, so nice win win.